VMWare/vmware-vcenter/ingest/parser.yml

```
@@ -101,6 +101,7 @@ stages:
 
       - set:
           host.ip: "{{parsed_json.message.client}}"
+          source.ip: "{{parsed_json.message.client}}"
```

After discussion with the Support team, we agree to extract the ip address to `source.ip` and remove `host.ip` that was introduced previously.
```suggestion
          source.ip: "{{parsed_json.message.client}}"
```